composables/use-log-event.ts

```diff
@@ -23,3 +26,71 @@ export default function useLogEvent(eventName: string, eventParams?: EventParams
     }
   }
 }
+
+export function useLogSetUser(user: {
```

Comment:
Why not move this composable to a separate file i.e. `use-set-log-user.ts`?
```suggestion
export function useSetLogUser(user: {
```

Reply:
these functions have similar structure
```
if... setSentry...
if... setGa...
if... setMeta...
if... setCrisp...
```
seems placing in same file is easier to maintain
renaming file to `use-logger` to avoid confusion